.github/pull_request_template.md

```diff
@@ -4,18 +4,18 @@ Describe in detail the changes you are proposing, and the rationale.
 
 See the contributing guide:
 
-https://github.com/placeholderplaceholderplaceholder/opentf/blob/main/.github/CONTRIBUTING.md
+https://github.com/opentffoundation/opentf/blob/main/CONTRIBUTING.md
```

Comment:
We are keeping this as `placeholderplaceholderplaceholder` for now, even though the link is broken, because the organization name is not finalized yet, and it'll help us replace it in the future